tools/accuracy_checker/accuracy_checker/annotation_converters/criteo_kaggle_dac.py

```diff
@@ -0,0 +1,116 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from pathlib import Path
+import numpy as np
+
+from ..representation import OAR3DTilingSegmentationAnnotation, ClassificationAnnotation
+from ..config import NumberField, StringField, PathField, BoolField
+from .format_converter import BaseFormatConverter
+from .format_converter import ConverterReturn
+
+class CriteoKaggleDACConverter(BaseFormatConverter):
+
+    __provider__ = 'criteo_kaggle_dac'
+    # annotation_types = (OAR3DTilingSegmentationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'testing_file': PathField(description="Path to testing file."),
+            "batch": NumberField(optional=True, default=128, description="Model batch"),
+            "subsample_size": NumberField(optional=True, default=0,
+                                           description="Limit total record count to batch * subsample size"),
+            "validation": BoolField(optional=True, default=True,
+                                           description="Allows to use half of dataset for validation purposes"),
+            "separator": StringField(optional=True, default='#',
+                                     description="Separator between input identifier and file identifier"),
+            "preprocessed_dir": PathField(optional=False, is_directory=True, check_exists=True,
+                                          description="Preprocessed dataset location")
+        })
+
+        return parameters
+
+    def configure(self):
+        self.src = self.get_value_from_config('testing_file')
+        self.batch = self.get_value_from_config('batch')
+        self.subsample = self.get_value_from_config('subsample_size')
+        self.validation = self.get_value_from_config('validation')
+        self.separator = self.get_value_from_config('separator')
+        self.preprocessed_dir = self.get_value_from_config('preprocessed_dir')
+
+    def convert(self, check_content=False, **kwargs):
+
+        preprocessed_folder = Path(self.preprocessed_dir)
+        input_folder = preprocessed_folder / "bs{}".format(self.batch) / 'input'
+        mask_folder = preprocessed_folder / "bs{}".format(self.batch) / 'mask'
+
+        _ = [folder.mkdir(parents=True) for folder in [input_folder, ] if not folder.exists()]
```

Comment:
here not need in list comprehension, usual cycle will be more suitable

Reply:
the same comment about you have always only one directory in cycle

Reply:
fixed